crates/driver/rustfmt.toml

```diff
@@ -0,0 +1,19 @@
+tab_spaces = 4
+edition = "2021"
+fn_single_line = false
+format_code_in_doc_comments = true
+format_strings = true
+imports_layout = "HorizontalVertical"
+imports_granularity = "One"
+normalize_doc_attributes = true
+reorder_imports = true
+reorder_impl_items = true
+group_imports = "StdExternalCrate"
+use_try_shorthand = true
+wrap_comments = true
+max_width = 100
+overflow_delimited_expr = true
```

Comment:
changes something that stable rustfmt already handles (but i'm fine with the change)